crypto/mbedtls/Kconfig

```diff
@@ -61,68 +63,64 @@ config MBEDTLS_ECP_RESTARTABLE
 
 config MBEDTLS_SELF_TEST
 	bool "Enable the checkup functions (*_self_test)."
-	default y
+	default n
 
 config MBEDTLS_SSL_DTLS_CLIENT_PORT_REUSE
 	bool "Enable server-side support for clients that reconnect from the same port."
-	default n
+	default y
 
 config MBEDTLS_BLOWFISH_C
 	bool "Enable the Blowfish block cipher."
-	default n
+	default y
 
 config MBEDTLS_CAMELLIA_C
 	bool "Enable the Camellia block cipher."
-	default n
-
-config MBEDTLS_CERTS_C
```

Comment:
why remove

Reply:
removed in v3.4.0 mbedtls_config.h has been removed in v3.4.0 mbedtls_config.h